### PR TITLE
Cleanup in util

### PR DESCRIPTION
**Pull request by oscargus**

### CODE REVIEW

File: src/main/java/net/sf/jabref/logic/util/io/FileHistory.java [hunk 1]
```diff
@@ -49,9 +49,9 @@ public int size() {
 
     public void newFile(String filename) {
         history.remove(filename);
-        history.addFirst(filename);
+        ((LinkedList<String>) history).addFirst(filename);
         while (history.size() > HISTORY_SIZE) {
-            history.removeLast();
+            ((LinkedList<String>) history).removeLast();
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/logic/util/io/FileHistory.java]: Hm, this looks strange.

**Comment by simonharrer** [comment]
Hm, it does not compile.

**Comment by oscargus** [comment]
Yeah, fixed now...

File: src/main/java/net/sf/jabref/logic/util/io/FileHistory.java (hunk 1 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/logic/util/io/FileHistory.java]: Indeed. Should one keep the definition as a LinkedList then? (I have to cast it to get access to removeLast().)

**Comment by simonharrer** [comment]
:+1:

File: src/main/java/net/sf/jabref/logic/util/io/FileHistory.java (hunk 1 above)
Comment (simonharrer) [inline src/main/java/net/sf/jabref/logic/util/io/FileHistory.java]: In that case, I would leave it at LinkedList as we make explicit use of this data structure.

**Pull request merged**